cpu/esp32/Makefile.include

```diff
@@ -129,6 +129,10 @@ ifneq (,$(filter esp_wifi_any esp_eth,$(USEMODULE)))
   CFLAGS += -DSCHED_PRIO_LEVELS=32
 endif
 
+ifneq (,$(filter lwip,$(USEMODULE)))
+  CFLAGS += -DTCPIP_THREAD_PRIO=2
+endif
```

Comment:
Also move the comment from `lwipopts.h` here so there is an explanation why this is needed / where this value is coming from.